### PR TITLE
Add control QueryType

### PR DESCRIPTION
**Pull request by jainxrohit**
Adding a control QueryType. This query type will be used to support transaction control and session control statement types.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by rschlussel — APPROVED** [review]
Looks good to me.  Get approval from @g1y before merging to release branch though.

**Review by g1y — APPROVED** [review]

**Pull request merged**
